### DISCUSSION TITLE
Sonoff SV GPIO as dry relay

### DISCUSSION THREAD

**Discussion by vaijab** [arendst/Tasmota#17867] [Q&A]
Hey. I am new to all of this. I wonder if it is possible to have a GPIO4 or GPIO5 configured as a dry relay?

I have managed to configure GPIO4 as a `Relay_i`, so I can now close it with a command, but when the relay is in `OFF`, there is voltage on S and G pins. Would it be possible to have zero voltage on those pins?

**Comment by sfromis** [comment]
For dry relay, use the "Isolated mode" wiring and small board modification. https://itead.cc/product/sonoff-sv/

**Reply by vaijab** [reply]
I have got this working, but that only gives me a single relay. I was thinking if it was possible to use GPIO4 and GPIO5 as dry relays too?

**Reply by sfromis** [reply]
No, a gpio cannot be used as a relay, as a relay makes/breaks a separate electrical connection able to carry a non-minor current. This is way outside of what a MCU chip can handle. What you can do depends on whether you actually need a relay.

**Reply by vaijab** [reply]
I see. Thanks for confirming that. What I need is a 2-channel dry switch to control my gates. There are two separate switches on my gate control panel, one for closing the gates and the other for opening. When either of those circuits is closed, the gate either opens or closes.

**Reply by barbudor** [reply]
There a lot of ESP+relay boards on AliExpress
Do NOT choose one with an Esp01 but one with a module such as Esp12 soldered on the board
There are different variant for various power supply

If you have any doubt, just share the link here for advices

**Reply by sfromis** [reply]
Here's an example of a board with two dry relays. Not tried it (thus not a specific recommendation), but to me it appears decent, and with easy options for adding pins for what may be useful of buttons or sensors. 
https://templates.blakadder.com/ESP12F_Relay_X2.html

You can also get a similar ESP32-based board, costs a bit more.
https://templates.blakadder.com/ESP32_Relay_X2.html
